nautobot/extras/plugins/views.py

```diff
@@ -30,6 +30,18 @@ def get(self, request):
         data = []
         for app in apps.get_app_configs():
             if app.name in settings.PLUGINS:
+                try:
+                    reverse(app.home_view_name)
+                except NoReverseMatch:
+                    app.home_view_name = None
+                try:
+                    reverse(app.config_view_name)
+                except NoReverseMatch:
+                    app.config_view_name = None
+                try:
+                    reverse(app.docs_view_name)
+                except NoReverseMatch:
+                    app.docs_view_name = None
```

Comment:
I'm not 100% comfortable with this actually modifying the `app` data in the error case. Might be safer/more correct to set `home_view_name`, etc. as local variables?

Reply:
Sure!